frontend/src/scenes/dashboard/Dashboard.tsx

```diff
@@ -110,7 +112,7 @@ function DashboardScene(): JSX.Element {
                 <EmptyDashboardComponent loading={itemsLoading} canEdit={canEditDashboard} />
             ) : (
                 <div>
-                    <div className="flex space-x-4 justify-between">
+                    <div className="dashboard-controls">
```

Comment:
We want to stick to utility classes these days as opposed to SCSS files. In fact, if you rebase on top of current `master`, you'll be able to use full Tailwind here. 😄 Including breakpoints, e.g. `md:flex-row` – though in this case perhaps flex wrapping would do the job?
```suggestion
                    <div className="flex gap-2 items-center justify-between flex-wrap">
```